zaza/openstack/charm_tests/ceilometer/tests.py

```diff
@@ -0,0 +1,118 @@
+        set_alternate = {'debug': new_value}
+        default_entry = {'DEFAULT': {'debug': [current_value]}}
+        alternate_entry = {'DEFAULT': {'debug': [new_value]}}
+
+        # Config file affected by juju set config change
+        conf_file = '/etc/ceilometer/ceilometer.conf'
+        services = {}
+        current_release = openstack_utils.get_os_release()
+        xenial_pike = openstack_utils.get_os_release('xenial_pike')
+        xenial_ocata = openstack_utils.get_os_release('xenial_ocata')
+        xenial_newton = openstack_utils.get_os_release('xenial_newton')
+        trusty_mitaka = openstack_utils.get_os_release('trusty_mitaka')
+        trusty_liberty = openstack_utils.get_os_release('trusty_liberty')
+
+        if current_release >= xenial_pike:
+            services = {
+                'ceilometer-polling: AgentManager worker(0)': conf_file,
+                'ceilometer-agent-notification: NotificationService worker(0)':
+                    conf_file,
+            }
+        elif current_release >= xenial_ocata:
+            services = {
+                'ceilometer-collector: CollectorService worker(0)': conf_file,
+                'ceilometer-polling: AgentManager worker(0)': conf_file,
+                'ceilometer-agent-notification: NotificationService worker(0)':
+                    conf_file,
+                'apache2': conf_file,
+            }
+        elif current_release >= xenial_newton:
+            services = {
+                'ceilometer-collector - CollectorService(0)': conf_file,
+                'ceilometer-polling - AgentManager(0)': conf_file,
+                'ceilometer-agent-notification - NotificationService(0)':
+                    conf_file,
+                'ceilometer-api': conf_file,
+            }
+        else:
+            services = {
+                'ceilometer-collector': conf_file,
+                'ceilometer-api': conf_file,
+                'ceilometer-agent-notification': conf_file,
+            }
+
+            if current_release < trusty_mitaka:
+                services['ceilometer-alarm-notifier'] = conf_file
+                services['ceilometer-alarm-evaluator'] = conf_file
+
+            if current_release >= trusty_liberty:
+                # Liberty and later
+                services['ceilometer-polling'] = conf_file
+            else:
+                # Juno and earlier
+                services['ceilometer-agent-central'] = conf_file
+
+        logging.info('changing config: {}'.format(set_alternate))
+        self.restart_on_changed(
+            conf_file,
+            set_default,
+            set_alternate,
+            default_entry,
+            alternate_entry,
+            services)
+
+    def test_901_pause_resume(self):
+        """Run pause and resume tests.
+
+        Pause service and check services are stopped then resume and check
+        they are started.
+        """
+        self.pause_resume(['ceilometer'])
```

Comment:
I don't think this is going to do anything. I think its supposed to be called as a context like [this](https://github.com/openstack-charmers/zaza-openstack-tests/blob/master/zaza/openstack/charm_tests/cinder/tests.py#L186)